scripts/jenkins/cloud/ansible/roles/ardana_qe_tests/tasks/process_test_results.yml

```diff
@@ -31,10 +31,21 @@
   failed_when: false
   register: test_results
 
+- name: Report '{{ test_name }}' results output if failed
+  debug:
+    var: test_results
+  when: test_results.rc != 0
+
 - name: Process test results from subunit
   set_fact:
     ardana_qe_test_results: "{{ ardana_qe_test_results | default({}) | combine({item.split()[0] | lower: item.split()[-1]}) }}"
   when: "'tests' in item"
   loop: "{{ test_results.stdout_lines }}"
   loop_control:
     label: "{{ item.split()[0] | lower }}: {{ item.split()[-1] }}"
+
+- name: Process test results from subunit
+  set_fact:
+    ardana_qe_test_results:
+      failed: 9999999
```

Comment:
why the magic number 9999999? Just curious.

Reply:
Makes it very very obvious that it is this special case and not a normal issue ;-)